pihole_adlist_tool

```diff
@@ -350,12 +351,22 @@ if [ "$UNIQUE" -eq 1 ];
         echo -e "  [i]  UNIQUE: Not shown"
 fi
 
+# do not RegEx when running in docker container:
+if [ "$PIHOLE_DOCKER" = 1 ]; then
+  REGEX_MODE=0
+else
+  REGEX_MODE=1
```

Comment:
It's still not working like this. Now it will enable RegEx mode on all non-docker installations regardless of the flag. You need to remove the `else` condition`

Reply:
🙈